### ISSUE TITLE
Can this be used on a Lenovo 500e chromebook 2nd gen?

### ISSUE THREAD

**Issue by poopsockman** [roblox-linux-wrapper/roblox-linux-wrapper#225]
uhh im using a chromebook with a good cpu which can run linux roblox stuff but idk which programs will allow me to run roblox on it, plus my chromebook is managed so i cant install via play store or adb debugging so does anyone know how to do this on chromebook or how to run roblox on bottles are any other possible software? if u do plz leave comment

**Comment by ntvmb** [comment]
Before I can help you any further, you need to make sure that your administrator has NOT disabled the Linux developer environment.

**Comment by poopsockman** [comment]
my school has blocked develop android apps aka adb debugging, but linux dev environment is allowed btw, also i tried it and it dosent really work but i need a tutorial on how to install apks on chrome os because i dont have a linux operating system and i need a way to do it without toggling the switch

**Comment by ntvmb** [comment]
Alright, then. You should be able to install Wine (and use RLW) on your Chromebook. To install Wine on your Chromebook, I'm going to refer you to [this guide](https://alvarotrigo.com/blog/wine-chromebook/).

**Comment by ntvmb** [comment]
Yep. You'll probably need to install Zenity as well, which you should be able to install simply by running `sudo apt install zenity`.

**Comment by poopsockman** [comment]
It gives me error saying no apps can open, find more in software its like the white screen with that page on it how do i fix?

**Comment by ntvmb** [comment]
You can try installing the Linux version of a browser of your choosing or it may be possible to run Roblox directly (e.x. `WINEPREFIX=~/.local/share/wineprefixes/roblox-wine wine "C:\\Program Files (x86)\\Roblox\\Versions\\version-72b95a99cfcf4b7d\\RobloxPlayerBeta.exe"`)

**Comment by poopsockman** [comment]
i have no idea what you just wrote but uhh how can i run that

**Comment by poopsockman** [comment]
sorry via terminal but how can i run roblox cuz i just leaves me with > sign

**Comment by ntvmb** [comment]
OK, I guess you should try using the Linux version of a web browser first, then try running Roblox directly via the terminal. Use the terminal to install the Linux version of web browsers (e.x. run `sudo apt install firefox` or `sudo apt install firefox-esr`)

**Comment by poopsockman** [comment]
alright so i did what you said but now when i press play nothing happens

**Comment by ntvmb** [comment]
You can try running this command to run Roblox directly: `WINEPREFIX=~/.local/share/wineprefixes/roblox-wine; wine "$(find "$WINEPREFIX" -name RobloxPlayerBeta.exe -print -quit)"`

**Comment by poopsockman** [comment]
got a memory dump

**Comment by ntvmb** [comment]
Ah, I just remembered: Roblox under Wine is broken right now. Try again later. You can check out the game's [AppDB page](https://appdb.winehq.org/objectManager.php?sClass=version&iId=15961) to get an idea of how Roblox under Wine is working at any given point.

**Comment by poopsockman** [comment]
ok thanks
